modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -4874,6 +4874,18 @@
 			"jsp"
 		]
 	},
+	{
+		"classNames": [
+			"Address",
+			"EmailAddress",
+			"OrgLabor",
+			"Phone",
+			"Website"
+		],
+		"from": "regex:(?<variableName>\\w+)\\.(?<prefixMethodName>get|set)Type",
+		"issueKey": "LPD-58265",
+		"to": "${variableName}.${prefixMethodName}ListType"
+	},
```

Comment:
@regiCesar21 it's equal to the LPD-36750 case. Only changing this one has the Id at the End of the methods. Wouldn't be better just edit that one and add the test cases on its file that already exists? Can link the new ticket to the previous one.